### PR TITLE
Feed page added

### PR DESCRIPTION
**Pull request by WRIOLtd**
https://github.com/webRunes/WRIO-InternetOS/issues/1581

### CODE REVIEW

File: src/base/components/Tabs.js [hunk 1]
```diff
@@ -17,10 +18,20 @@ import {
 import { findDOMNode } from 'react-dom';
 import EditExternal from 'CoreEditor/containers/EditExternal';
 const changeUrlToUrlForEdit = require('./utils/change_url_to_url_for_edit');
+import { connect } from 'react-redux';
+import { loadFeed } from '../../base/actions/actions';
+
+const feedURL  = 'https://muhammadumair0.github.io/feed/';
```
Comment (AlexeyAnshakov) [inline src/base/components/Tabs.js]: It's not a const, and must be taken from json-ld block

File: src/base/components/Tabs.js [hunk 2]
```diff
@@ -17,10 +18,20 @@ import {
...
 
 const HEADER_PADDING = 15; // variable set in CSS
```
Comment (AlexeyAnshakov) [inline src/base/components/Tabs.js:24]: No styles in JS. Delete

Comment (WRIOLtd) [reply src/base/components/Tabs.js:24]: I didnt added this

Comment (AlexeyAnshakov) [reply src/base/components/Tabs.js:24]: Yes, i know, ,missed that one before. Please delete

Comment (WRIOdev) [reply src/base/components/Tabs.js]: It was added by Michael Bilenko (2 years ago)

Comment (WRIOLtd) [reply src/base/components/Tabs.js:24]: ok

Comment (WRIOLtd) [reply src/base/components/Tabs.js:24]: removed

**Comment by AlexeyAnshakov** [comment]
Please add into the description an issue link. I've already added it as an example.

File: src/base/components/Tabs.js (hunk 2 above)
Comment (AlexeyAnshakov) [reply src/base/components/Tabs.js:24]: Please be sure there no "HEADER_PADDING" in the code anywhere.

Comment (WRIOLtd) [reply src/base/components/Tabs.js:24]: ok

File: src/base/actions/actions.js [hunk 3]
```diff
@@ -155,8 +155,12 @@ export function loadDocumentWithData(data: LdJsonDocument, url: string) {
     });
 
     toc.external.map(async (externalDoc: Object, i: number) => {
+      if(externalDoc.name == 'Vacancies') {
```
Comment (AlexeyAnshakov) [inline src/base/actions/actions.js]: there must be no linkage to the name.

Comment (WRIOLtd) [reply src/base/actions/actions.js]: ok

**Pull request merged**